<launch>
  <rosparam command="load" file="$(find wbc_pr2_ctrl)/config/pr2_wbc.yaml" />
  <node
      pkg="wbc_pr2_ctrl"
      type="opspace_servo"
      name="opspace_servo"
      args="-v -b $(find wbc_pr2_ctrl)/config/opspace_skills.yaml -t $(find wbc_pr2_ctrl)/config/tao_tree_pr2_base_without_torso_and_head.xml"
      output="screen">
  </node>
</launch>
